webapp/src/NextPlayerButton.js

```diff
@@ -17,9 +17,10 @@ class NextPlayerButton extends Component {
     }
 
     render() {
+        const label = ( this.props.game.Ongoing == 3 ? "Hold" : "Next" )
```

Comment:
This should be inverted. It should display 'Hold' when playing and 'Next' while on hold